modules/giapi/src/main/scala/giapi/client/ghost/GHOSTClient.scala

```diff
@@ -0,0 +1,53 @@
+package giapi.client.ghost
+
+import cats.implicits._
+import fs2.Stream
+import giapi.client.{Giapi, GiapiClient}
+
+import scala.concurrent.duration._
+
+/**
+  * Client for GHOST
+  */
+class GHOSTClient[F[_]](override val giapi: Giapi[F]) extends GiapiClient[F] {
+  // TODO: Should this be moved to the GIAPIClient? Or will GHOST have a specific value?
```

Comment:
Good question, I think this is defined for GIAPI so we could move it to `GiapiClient`